Manager/BackupManager.php

```
@@ -71,17 +71,8 @@ public function execute()
             // write log
             $this->logger->critical($e);
 
+            //Should we throw a general exception here? Maybe we should return a bool.
```

So you prefer to throw the exception maybe from the command?